pkg/gofr/context.go

```diff
@@ -28,6 +31,12 @@ type Context struct {
 	responder Responder
 }
 
+type AuthInfo interface {
+	GetClaims() jwt.MapClaims
+	GetUsername() string
```

Comment:
I would suggest this "username" is term made of two words

```suggestion
	GetUserName() string
```

Reply:
I feel username is a single word , so it should not be UserName.